learners/setup.md

```diff
@@ -76,7 +85,7 @@ Now that you have the environment set up, let's download the necessary files fro
 
 ### Option A: manual download
 
-1. Open your web browser and navigate to [this lesson's data Zenodo record address](https://zenodo.org/records/13311246).
+1. Open your web browser and navigate to [this lesson's data Zenodo record address](https://zenodo.org/records/13929407).
```

Comment:
I think you created a new zenodo entry, but it's better to just update the already existing one: https://zenodo.org/records/13311246. I just added you as collaborator so you should be able to edit and publish a new version there. Then delete the duplicated record (https://zenodo.org/records/13929407)

Reply:
OK, I made a new version of the record, as you must create a new version (and therefore different record number but same DOI) to modify edit or delete files.